packages/datagateway-search/src/table/datafileSearchTable.component.test.tsx

```diff
@@ -4,18 +4,17 @@ import { initialState as dgSearchInitialState } from '../state/reducers/dgsearch
 import configureStore from 'redux-mock-store';
```

Comment:
This file will need a refactor with RTL

Reply:
I migrated this test to RTL in #1447. Should I port the test to this PR?

Reply:
Nah, eventually this PR and yours will need to sync and I'd rather do that all in one go. Good it's been resolved, thanks!